test/test_image.py

```diff
@@ -196,19 +233,6 @@ def test_write_png(self):
 
             self.assertTrue(img_pil.equal(saved_image))
 
-    def test_decode_image(self):
```

Comment:
One can argue that this test is useful because it covers directly the `decode_image()` method. Unfortunately it's flaky and to fix it, I would have to copy-paste the mitigations done on `test_decode_jpeg()`. I decided to remove it because this method is just a wrapper for `decode_jpeg()` and `decode_png()` which are exhaustively tested.

Reply:
> Also, can you expand why this is flaky?

For the reasons you covered [here](https://github.com/pytorch/vision/pull/2988#discussion_r522983218). It would require the same mitigation with tolerance I did there. 

Happy to restore it, just note that there will be some duplication of code. An alternative is to replace the use of `decode_jpeg()` with `decode_image()` in the `test_decode_jpeg` unit-test. The code-path will be covered and it will be clear if it breaks what's the issue. Which one do you prefer?

Reply:
Applied my proposed change on latest commit.

Reply:
ok. I would still have preferred to keep each function separately testing only one function, as now the `test_decode_*` functions exercise the codepath of two different C++ functions. But the previous tests were not perfect either, so instead of being blocked by this let's move forward